### ISSUE TITLE
PUI English errors

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/csl-corrections#41]
@AnnaRybakovaT 

Let's try that 'Process 2' approach (see #31) for pui  (purana index).

1.  Be sure your fork of csl-orig is up to date
2.  I see that as of now, in https://github.com/AnnaRybakovaT/csl-orig,
    `This branch is 1 commit ahead, 2 commits behind sanskrit-lexicon:master.`

  This syncing step is probably the hardest step.  
  

Once syncing is done,
Work with local copy of 
* csl-orig/v02/pui/pui.txt
* csl-orig/v02/pui/misc/pui_error.txt
   * This was copied from https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/pui_error.txt

Once you've done the corrections:
* Probably your fork will again be behind sanskrit-lexicon/csl-orig  (since this repository gets lots of
  changes).  You may need to do another sync before next step.  I'm not sure.

* create pull request 

Then, the ball will be in my court (or Dhaval's) to deal with the pull request.

**Comment by gasyoun** [comment]
> Then, the ball will be in my court (or Dhaval's) to deal with the pull request.

Let it be @drdhaval2785 court, let him be the gatekeeper, please.

**Comment by drdhaval2785** [comment]
I am happy to be gatekeeper for github repo. But Jim will have to do everything on Cologne servers. My account there has expired.

**Comment by AnnaRybakovaT** [comment]
> 1. Be sure your fork of csl-orig is up to date

I need your advice, since I am in doubt what to do next 

![image](https://user-images.githubusercontent.com/74726889/104045896-482f6a80-51f0-11eb-9a05-216df365de07.png)

I suppose I should do something with "write access"

![image](https://user-images.githubusercontent.com/74726889/104045973-6a28ed00-51f0-11eb-8fea-038f7a7502fd.png)

**Comment by AnnaRybakovaT** [comment]
Or I did something wrong?

**Comment by drdhaval2785** [comment]
I am not sure, but it seema that you did the reverse thing. 

> Be sure your fork of csl-orig is up to date

You need to create a pull request on AnnaRybakovaT/csl-orig from sanskrit-lexicon/csl-orig to update your fork.

**Comment by drdhaval2785** [comment]
https://github.com/sanskrit-lexicon/csl-corrections/issues/31#issuecomment-754429957

**Comment by AnnaRybakovaT** [comment]
> I am not sure, but it seema that you did the reverse thing.

I also this noticed.
Probably, I undestood the problem. Will try one more time.

**Comment by drdhaval2785** [comment]
Most crucial step is `4. Click on 'switching the base'` 

If that is missed, it may lead to such problem.

**Comment by AnnaRybakovaT** [comment]
There is something wrong from the beginning.

- I see 3 commits behind
- click "Pull request"
![image](https://user-images.githubusercontent.com/74726889/104049190-9bf08280-51f5-11eb-9d53-943575a8e494.png)

and after it looks like I wanna to merge into sanskrit-lexicon (but it is wrong) and of course I don't have 'switching the base'

![image](https://user-images.githubusercontent.com/74726889/104049696-6e580900-51f6-11eb-9caf-39b579714875.png)

**Comment by drdhaval2785** [comment]
Ok. Leave it for now . I am going off to sleep. Will see tomorrow morning. Please bear with me. It is midnight in India

**Comment by AnnaRybakovaT** [comment]
Dear Dhaval,
I didn't want to bother you. I am so sorry.
Good night!

Will find solution tomorrow.

**Comment by gasyoun** [comment]
> everything on Cologne servers. My account there has expired.

@funderburkjim whom to write at Cologne?

**Comment by funderburkjim** [comment]
> whom to write at Cologne?

I think Dhaval knows who.  He can email me otherwise.

**Comment by funderburkjim** [comment]
@AnnaRybakovaT 

In this circumstance,  one solution would be to delete your fork:  https://github.com/AnnaRybakovaT/csl-orig

This can be done in browser interface:
* go to https://github.com/AnnaRybakovaT/csl-orig in browser
* choose settings
* Go to bottom of page, and choose 'delete repository'

On your local machine, you could either save local csl-orig under a new name
or delete local csl-orig

Then, you can again fork https://github.com/sanskrit-lexicon/csl-orig, clone to your machine, etc. etc.

But you may prefer to work with Dhaval to get another solution.

**Comment by AnnaRybakovaT** [comment]
> But you may prefer to work with Dhaval to get another solution.

Dear Jim, 
Thanks a lot for your support. 
I suppose - better to wait for a Jim's feed-back and after I will try to solve this issue. 
In any case I did a copy of pui_error.txt and I started analyzing.

**Comment by AnnaRybakovaT** [comment]
> * You may need to do another sync before next step.  I'm not sure.

Dear Jim,
I have one, maybe silly qestion, but I wanna to be sure for avoiding future mistakes.
So - I corrected my local copies of pui_error.txt and pui.txt and if before pull request I have to do syncing, I worry if my corrected local files could be updated according to the pero, where exist only  initial files pui_error.txt and pui.txt whithout corrections..

If it can be - of course before syncing I can save copies of corrected files (somewhere out of my local repo). I just would like to understand - there is sense to do it or not.

**Comment by AnnaRybakovaT** [comment]
Now 1/3 of work has done. In couple of days I will be ready to make pull request of the whole list of errors.

**Comment by drdhaval2785** [comment]
@AnnaRybakovaT as we are doing mass pull request for the first time, I would advise to store the corrected files and notes somewhere else, outside the repo. If we bungle anything, we do not want to loose your hard work.

**Comment by drdhaval2785** [comment]
>  I worry if my corrected local files could be updated according to the pero, where exist only initial files pui_error.txt and pui.txt whithout corrections..

I dont think this would happen.
There is only one rare event, which may do so. If there are changes in the same line of pui.txt on the sanskrit-lexicon/csl-orig repo and your repo, there will be a problem. Git would not know what to keep and what to discard. It will be on our shoulders to decide that. 
But in that case 'Able to merge' message would nog come. It will say 'There is a conflict' kind of message.

**Comment by funderburkjim** [comment]
>  I can save copies of corrected files (somewhere out of my local repo).

@AnnaRybakovaT   When I was getting used to git,  I obsessively saved copies my work outside of the
repo,  so I would have my work in case I goofed with git.   With experience,  this need to save is less.
So I commend your conservative instinct to save work.

**Comment by AnnaRybakovaT** [comment]
Dear all,
Many thanks for your comments and advices.

**Comment by AnnaRybakovaT** [comment]
> Once you've done the corrections:
> 
> * Probably your fork will again be behind sanskrit-lexicon/csl-orig  (since this repository gets lots of
>   changes).  You may need to do another sync before next step.  I'm not sure.
> * create pull request

Dear all, 
Could I clarify something? 

Is this algorithm correct:

1) The correction of PUI is ready

2) My fork is behind sanskrit-lexicon/csl-orig and I have to make syncing

3) I have to make the Step 6
![image](https://user-images.githubusercontent.com/74726889/104326244-90090700-54fa-11eb-8d5f-3b1481836398.png)

4) And the Step 7
![image](https://user-images.githubusercontent.com/74726889/104326371-bb8bf180-54fa-11eb-8dc4-e3fd9dd3ed4a.png)

**Comment by drdhaval2785** [comment]
As your fork is behind the original repo, correct order would be the following.

1. https://github.com/sanskrit-lexicon/csl-corrections/issues/31#issuecomment-754429957
2. https://github.com/sanskrit-lexicon/csl-corrections/issues/31#issuecomment-754444135
3. https://github.com/sanskrit-lexicon/csl-corrections/issues/31#issuecomment-753750649
4. https://github.com/sanskrit-lexicon/csl-corrections/issues/31#issuecomment-753752742

Kindly give it a try and let me know.

**Comment by AnnaRybakovaT** [comment]
> Kindly give it a try and let me know

Dear Dhaval,
Thanks a lot!
Will try)))

**Comment by AnnaRybakovaT** [comment]
Dear Dhaval,
I have difficulties on the step 7 
I created the pull reguest but I can't merge

![image](https://user-images.githubusercontent.com/74726889/104337632-c0ef3900-5506-11eb-8117-5362173d4a79.png)

**Comment by drdhaval2785** [comment]
@AnnaRybakovaT 
You have done everything properly. There is nothing remaining to be done from your side.

You can't merge in sanskrit-lexicon/csl-orig, because you are not the owner of that repository. 

It will be a duty of Jim / me to review your corrections and merge them to sanskrit-lexicon/csl-orig repository.

Well done. You submitted your corrections via pull request. I see your pull reuest . 

@funderburkjim I want to give it a try to review the changes in PUI, and merge.

Anna, you may continue with dictionary of your choice for next batch.

**Comment by gasyoun** [comment]
> Anna, you may continue with dictionary of your choice for next batch.

Hurray. I'm proud of Anna.

**Comment by AnnaRybakovaT** [comment]
> Anna, you may continue with dictionary of your choice for next batch.

Dear all,
I let you to make the choice. I will be happy to work with any dictionary))) 
I am waiting for your next task.

**Comment by gasyoun** [comment]
> I let you to make the choice

Here it comes:

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/pe_error.txt

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/yat_error.txt

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/shs_error.txt

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/cae_error.txt

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/mw72_error.txt

**Comment by drdhaval2785** [comment]
Completed the review of the work.
My comments added .

> 274:aRqam:egglike: OK
585:apasvAnta:loveable: OK
; alternative form is "lovable"
588:apANkteya:interdining: OK (to join in a common meal)
1197:AtatAyin:seditionists: OK (one who engages in sedition)
1398:Arza:doab: OK (in South Asia - the land lying between two confluent rivers)
; similar to "interfluve"
1570:indrARI:kuladevatas: OK (Sanskrit term)
; Kuladevatā - a tutelary deity, the guardian deity of a family
; Dhaval : Should it not be Kuladevatās ? Missing diacritic mark?
1640:IhASila:dualities: OK
2462:kapAlitvam:theophany: OK (a visible manifestation of a deity)
2734:kaSyapapadam:perfomance: performance
; scan error
2842:kAmyA:daughtes: daughters
; scan error
2916:kAlanemI:abhastalam: gabhastalam
; Scan error. I corrected like in 3100:kuYjara. Please - recheck
; Dhaval: I checked it with kuYjara. It shows `fourth tala or gabhastalam`. Therefore, We have to mark this entry in kAlanemI as print error.
3171:kumAra:resplendant: OK
; Obsolete, modern spelling is "resplendent"
3252:kurava:presious: precious
; scan error
3506:kfzRa:stupifying, embarassment: stupefying, embarrassment
; scan error
; a link to the scan page 346 [Page1-436 + 43] doesn't work
; Dhaval: Both items are typed as they are in the scan. But they are wrong words as per English. Therefore, marking it as print change.
3507:nAmakaraRam:seagod,topcrest,blindmans,externment,invisilability,aIb
; links to the scan pages 346-350, mentioned here, don't work. 
; I can't check scan errors
3507:nAmakaraRam:seagod: sea-god
; Dhaval: reverted to seagod as per scan.
3507:nAmakaraRam:topcrest: top crest
; Dhaval: reverted to topcrest as per scan.
3507:nAmakaraRam:blindmans: OK (blindman's buff)
3507:nAmakaraRam:externment: OK (banishment)
3507:nAmakaraRam:invisilability: inviolability
; I am not sure. I don't see the scan.
; Dhaval: I am also not sure. inviolability seems right in the context. Changing to print change.
3507:nAmakaraRam:aIb: OK ( a)Ib.)
; Maybe to put a gap " " between ")" and "Ib." There are some similar lines with gapes, some without.
; Dhaval: Noted the issue at https://github.com/sanskrit-lexicon/csl-corrections/issues/44
3631:kESika:caidyas: OK (Sanskrit word - descendants of Cedi)
3833:kzatriya:bX: OK (we see -  b)X )
3865:kzIroda:yoganidra: OK (Sanskrit term)
; Dhaval: yoganidrā . Print error
4024:ganDa:gandhamodavaha: OK (Sanskrit word)
4039:ganDarva:semidivine: OK
4078:gayAsura:perforhmed: performed
; scan error
4233:guRa:prakrti: prakṛti
; "ṛ" in scan - is not clear
; Dhaval: It is there. But quite smudged.
4317:goDarma:becme: became
; scan error
4457:grAmyAraRya:vritri: OK (Sanskrit word)
;Dhaval: It seems to be a print error for vrIhi. Debatable. 
4521:cakravartin:rathakrit: OK (Sanskrit word)
4632:candramaRqalam:degresses: OK
4840:cidvahni:hotri: OK (Sanskrit word)
4993:jambU:roseapple: rose apple
; scan error
; Dhaval: print change.
5057:jayaviGnam:stupefication: stupefaction
; scan error
; Dhava: print change.
5250:jYAnam:aVi: OK ( 2a)Vi.)
5342:taptakumBa:gaolers: OK
; more common spelling is "jailers"
5828:dandaSUka:serpant: serpent
; scan error
; Dhaval: print change.
6068:durga:outways: OK
; rare, synonyms "ways out", "exits"
6302:devahUtI:stupour: OK
; Obsolete, a modern form is "stupor"
6427:dvAdaSAkzara:medidate: meditate
; scan error
; Dhaval: Whenever Anna has noted 'scan error', it is supposed to be read as 'print change'
6563:DarmayAga:mudhouses: mud houses
; scan error
6576:DarmavratA:despitc: despite
; in the scan "e" is not clear
6930:narakam:bhaumam: OK (Sanskrit word)
6949:narmadA:austereties: austerities
; scan error
6987:nahuza:efflulgence,overweaning: effulgence, overweening
; scan error
7285:nIdAGa:thousannd: thousand
; scan error
7371:nErfta:valourous: OK
; more common form is "valorous"
7392:nyAsa:eg,gairikam: OK (1. abbreviation, 2. a Sanskrit word)
7438:paYcamuKa:sthambha: OK (Sanskrit word)
7505:patram:likhitam: OK (Sanskrit word)
7641:parIkzit:detatched: detached
; scan error
7774:pApA:trafficing: trafficking
; scan error
7795:pArijAta:izens: citizens
; in the scan "t" is invisible
7884:piRyAkam:ingudi: OK (tree)
7934:piSAcA:professionalists: OK
7981:puRqram:snowfed: snow-fed
; scan error
8229:pUraRa:plumpy: OK (from "plump")
; Please double check, since this meaning is strange in context of the sentence
; Dhaval: Seems to denote its plump look. Keep as it is.
8360:pEla:flating: floating (I am not sure, please, recheck)
; probably - scan error
; Dhaval: flate is not a proper word. floating seems fine.
8369:potriRI:cakraratha: OK (Sanskrit term)
8441:prakfti:acquiesed: acquiesced
; scan error
8587:prabala:attendent: attendant
; scan error
8705:prahlAda:nector,poohpoohed,predicetd: nectar, pooh-poohed, predicted
; links to the scan pages 435-437, mentioned here, don't work
; probably scan errors (I don't see the scan)
; Dhaval: All three are print changes.
8772:prAyopaveSa:pratcised: practised
; scan error
8780:prAsAda:descrbied: described
; scan error
8920:balarAma:sufference: sufferance
; links to the scan pages 464-466, mentioned here, don't work
; probably scan errors (I don't see the scan)
; Dhaval: print change.
8942:bali:protcction: protection
; "e" in scan - is not clear
8954:balipAtra:gmelina: OK (plant)
8954:balipAtra:arborca: arborea
; "e" in scan - is not clear
8954:balipAtra:jonesia: OK (plant)
9076:buDa:constellar: 
; I didn't find "constellar" in any English dictionaries. 
; maybe it is an obsolete form of "constellational", "constellatory"
; I didn't correct pui.txt
; Dhaval: Agree. Leave as it is.
9136:bfhadukTa:tripravara: OK (Sanskrit term)
9188:bfhadvanam:neghbouring: neighbouring
; scan error
9229:brahmaja:youthhood: OK
; synonym "youth"
9302:brahmA:subtlity: subtlety
; links to the scan pages 515-516, mentioned here, don't work
; scan error probably (I don't see the scan)
9328:brAhmaRa:himseslf: himself
; scan error
9342:brAhmI:kalpalata: OK (Sanskrit word)
; Dhaval: kalpalatā. Print change
9452:BadrASva:excell: excel
; scan error
9535:BAgavata:essense: essence
; scan error
9585:BAratAKyAnam:cssence: essence
; "e" in scan - is not clear
9630:Bikzu:sangha: OK (Sanskrit word)
9630:Bikzu:resig: resig-
; Scan error, "-" is invisible ("resignation" is hyphenated)
9926:maRqapa:tapovanam: OK (Sanskrit term)
9958:matsya:trible: tribe (I am not sure, better to recheck)
; Scan error
; Dhaval: It is a typo. The scan has 'tribe'. Digitization was done wrongly to 'trible'.
10354:mahAn:sankalpa: OK (Sanskrit word)
10526:maheSvara:angas: OK (Sanskrit word)
10550:mAGa:saptami: OK (Sanskrit word)
10689:mAruta:incure: incur
; Scan error
10706:mArga:uparathyas: OK (Sanskrit word)
; but mayby upārathyas (in this case - scan error)
; Dhaval: Keep uparathyas.
11144:yajYa:vikarika: OK (Sanskrit word)
11145:yajYa:balikarma: OK (Sanskrit term)
11149:yajYadravya:pavitram: OK (Sanskrit word)
11163:yajYaSrI:karnika: OK (Sanskrit word)
11180:yadu:sannyasin: OK (Sanskrit word)
11222:yayAti:punyam: puṇyam
11351:yogi:ahimsaka: OK (Sanskrit word)
11584:rAjasUyam:hota: OK (Sanskrit word)
; but mayby hotā (in this case - scan error)
; Dhaval: This is a generic issue in PUI. Noted at https://github.com/sanskrit-lexicon/csl-corrections/issues/45 .
11667:rukmapArijAtavana:blackman: OK
11727:ruDirAnDa:begger: beggar
; Scan error
11769:revatI:daugter: daughter
; Scan error
11819:rohiRI:deceuse: decease
; "a" in the scan - is not clear
11848:rOrava:incendiarism: OK
11957:lAvaRyavati:ayatam: OK (Sanskrit word)
; Dhaval: ayatam is not a Sanskrit word. ayutam is. Print change. to 'ayutam'.
12291:varRASrama:homekeepers: home-keepers
; Scan-error, synonym "householders"
12603:vAruRakratu:sthopa,devakanyas: OK (Sanskrit words)
12925:vinAyaka:enragement: OK
13247:viSvasPAni:enuch: eunuch
13288:vizaGnAni:antedotes: antidotes
; Scan error
13426:vfka:whosesoever: OK
13461:vftra:dharmic: OK
13628:vEkArika:digdevatas: OK (Sanskrit term)
13709:vEBrAja:quartzite: OK (rock)
13772:vESvadeva:chamsi: OK (Sanskrit word)
; Better to recheck
; Dhaval: No idea. Leave as it is.
13846:vyucCinna:avyucchinna: OK (Sanskrit word)
13997:Satakratu:anadhyaya: OK (Sanskrit word)
14135:Sambara:bady: baby
14297:SAnti:hetrodoxy: heterodoxy
; Scan error
14445:Siva:receving: receiving
; links to the scan pages 422-423, mentioned here, don't work
; Maybe - Scan error (I don't see the scan)
14462:Sivapuram:utpalam: OK (Sanskrit word)
14662:SuBalakzaRa:devasvastika: OK (Sanskrit word)
14681:SUdra:panktibhojanam: OK (Sanskrit word)
14740:Seza:venomed: OK
; Archaic
14800:SOcam:mlecchahood: OK
; synonym "barbarism"
14879:SrAdDam:hemophrodites,hemophrodite: hermaphrodites, hermaphrodite
; links to the scan pages 477-478, mentioned here, don't work
; Maybe - Scan error (I don't see the scan)
; Dhaval: print change.
14931:SrIvanam:bilvavanam: OK (Sanskrit word)
15123:saMgraha:muhurta: OK (Sanskrit word)
15193:satyam:tapolokam: OK (Sanskrit word)
15243:satrAjita:seperate: separate
; Scan error
15326:sannati:repentence: repentance
; Scan error
15418:samudrasena:horsefaced: horse-faced
;  Scan error, BUT maybe "horsefaced" is also correct
; Dhaval: horse-faced kept.
15502:saMsAra:duhham: OK (Sanskrit word)
15543:sarga:rudimental: OK
15546:sarpa:nabhonabha: OK (Sanskrit word)
15897:sidDi:vaivastya: OK (Sanskrit word)
16020:sugrIva:satsanga: OK (Sanskrit word)
16091:sudakziRa:vengence: vengeance
; Scan error
16636:sUkara:stealers: OK (larcenists, thiefs)
16640:sUkzmam:indiriyas: OK (Sanskrit word)
; Dhaval. Print change to indriyas. Removed extra i.
16845:sOBAgyazwakam:kurumbham: OK (Sanskrit word)
16920:stamBa:octogonal: octagonal
; "octogonal" (which is in the scan) is also correct but it is an obsolete form
16920:stamBa:dvivajra,bakulu: OK (Sanskrit words)
16947:sTalapadmini:anantasada: OK (Sanskrit word)
16999:sva:divaloka: OK (Sanskrit word)
17192:hari:indwellers: OK
17279:havi:havyakavyam: OK (Sanskrit word)
;EXTRA
9136:bfhadukTa:An: An- 
; An-giras, "-" in the scan is invisible

**Comment by drdhaval2785** [comment]
This dictionary is too erratic in diacritics. Also, printing errors are also myriads. Therefore, hard effort is not made to mark all print changes religiously.

**Comment by gasyoun** [comment]
> too erratic in diacritics + printing errors are also myriads

Maybe we should have such data for each one? Let's say printing quality.

**Comment by funderburkjim** [comment]
> Dhaval: print change.

@drdhaval2785  Are you going to add to https://github.com/sanskrit-lexicon/csl-corrections/blob/master/dictionaries/pui/pui_printchange.txt ?

**Comment by drdhaval2785** [comment]
Yes. Will transfer print changes to this file today.

**Comment by AnnaRybakovaT** [comment]
> Here it comes:
> 
> https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/pe_error.txt

Dear all,

I suggest to start from the beginning of this list - from Puranic Encyclopedia

**Comment by funderburkjim** [comment]
@AnnaRybakovaT   Go for it!  Looks like a good choice.

Are you comfortable now with 'process2'  (the one with forked csl-orig?)

**Comment by AnnaRybakovaT** [comment]
> Are you comfortable now with 'process2' (the one with forked csl-orig?)

Yes, I don't see any difficulties. In any case I have all instructions and your support.

How I see - the Puranic Encyclopedia consists around 300 potential errors. Should I create Pull Request: 
a) when I finish analysing the all list of errors? 
b) or I can make some Pull Requests?

**Comment by drdhaval2785** [comment]
B is better.
Let us keep it around 100 entries per pull request.

**Comment by gasyoun** [comment]
> Let us keep it around 100 entries per pull request.

Right.

**Comment by AnnaRybakovaT** [comment]
> Let us keep it around 100 entries per pull request.

OK. Thanks!

**Comment by AnnaRybakovaT** [comment]
> Let us keep it around 100 entries per pull request.

Dear all,

I am ready to make the 1st Pull Request.
Could you create for me the file pe_error.txt?
Or I can try to push my file with errors (where I copied the all list of errors) to - https://github.com/sanskrit-lexicon/csl-orig/tree/master/v02/pe ?

**Comment by funderburkjim** [comment]
@AnnaRybakovaT 

I can think of 4 ways to deal with pe_error.txt.

## Method 1
I've added corrections team to the sanskrit-lexicon/CORRECTIONS  repository --  this is the repository where Dhaval
put the original pe_error.txt, in the english_errors/output folder.

https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/english_error/output/pe_error.txt

Since you are a member of this team, and the members of this team can write to the repository,

You should now be able to (in browser), edit  the pe_error.txt file, replacing it with your revised file (copy/paste), and commit the revision.

## Method 2
1.  clone (not fork) the sanskrit-lexicon/CORRECTIONS repository
2. Put your modified pe_error.txt where it belongs  in your local repository
3. git add, git commit, git push   to get the updated pe_error.txt file

## Method 3
drag your local copy of pe_error.txt into a comment here in this issue.

## Method 4
1. fork sanskrit-lexicon/CORRECTIONS
2. put your revised pe_error.txt file where it belongs  in your forked local repository
3. make pull request.

Probably method 1 or 3 is simplest.  Just let know where to look for the revised pe_error.txt.

**Comment by drdhaval2785** [comment]
Method 2 is the way to go

**Comment by AnnaRybakovaT** [comment]
> 2\. Put your modified pe_error.txt where it belongs  in your local repository

Dear all, 
thanks for your comments!

According to the method 2 - I cloned (not fork) the sanskrit-lexicon/CORRECTIONS repository.
This repository has a lot of folders. Where exactly I should put my modified pe_error.txt ?
Probably to this folder:
C:\Users\Rybakova\Documents\sanskrit-lexicon\CORRECTIONS\english_error\output ?
(In this case I should use the name of my  modified file - something like pe_error1.txt  )

**Comment by funderburkjim** [comment]
Yes, in english_error/output/

I would use pe_error.txt   --- same name as it started with.  That way, we will be able to see that pe_error.txt has been modified.

**Comment by gasyoun** [comment]
> same name as it started with. That way, we will be able to see that pe_error.txt has been modified.

Exactly.

**Comment by AnnaRybakovaT** [comment]
> Yes, in english_error/output/

Many thanks!!
It has done.

As I understand - now I should follow the process2 (with forked csl-orig) to  create pull request for modified pe.txt?

**Comment by AnnaRybakovaT** [comment]
Dear all,
I have just created pull request for the Batch 1- pe.txt. 
I hope, everyhing is ok.

**Comment by gasyoun** [comment]
> I hope, everyhing is ok.

No, it's not ok, it's perfect.

![piace](https://user-images.githubusercontent.com/80761/105762070-f2372280-5f64-11eb-8f86-ee5d3972a36f.JPG)

**Comment by AnnaRybakovaT** [comment]
Dear all,
May I create pull request for the Batch 2 - pe.txt ?

**Comment by drdhaval2785** [comment]
Sure. Go ahead. 

This will also give us some homework. What to do when the previous Pull Request has not been merged, and a new pull request for the same file occurs. I guess git would have this problem covered. Should not be an issue.

**Comment by gasyoun** [comment]
> I guess git would have this problem covered.

That's what I believe as well.

**Comment by AnnaRybakovaT** [comment]
Dear all,
I need your advice. I can't push the file pe_errors.txt. 
Should I update sanskrit-lexicon/CORRECTIONS by "git pull" ?

![image](https://user-images.githubusercontent.com/74726889/107142988-892bb380-6943-11eb-9452-202f17ea88a0.png)

**Comment by drdhaval2785** [comment]
Yes. Update by 'git pull'

**Comment by drdhaval2785** [comment]
It would be advisable to copy one copy of pe_errors.txt somewhere on your computer, as a backup if something goes wrong.

**Comment by AnnaRybakovaT** [comment]
> It would be advisable to copy one copy of pe_errors.txt somewhere on your computer, as a backup if something goes wrong.

Dear Dhaval,
Thanks a lot!

**Comment by AnnaRybakovaT** [comment]
Now I have this message:

![image](https://user-images.githubusercontent.com/74726889/107143748-d9a51000-6947-11eb-8fe4-b24e05db275a.png)

**Comment by drdhaval2785** [comment]
We need to agree to it.

1. If Enter works, press enter.
2. else, try `:wx`
3. or Ctrl+C

**Comment by drdhaval2785** [comment]
Sorry.
It is `:wq` 
The vim way of saying 'Save and Quit the file.

**Comment by AnnaRybakovaT** [comment]
Thanks!
The updated file pe-error.txt is pushed.

**Comment by AnnaRybakovaT** [comment]
As well I have  uploaded the change to my copy of csl-orig (Step 6).

But I can't  make the new pull request (Step 7). 
As you see I don't have 'Create Pull Request' button

![image](https://user-images.githubusercontent.com/74726889/107144897-4079f780-694f-11eb-880a-c07d8d6de21a.png)

**Comment by drdhaval2785** [comment]
I think you will not have to open new pull request. The changes got reflected in your ongoing pull request.

See https://github.com/sanskrit-lexicon/csl-orig/pull/467/commits

Where your two commits of batch 2 which are posted 1 hour ago are also shown. 

So when we merge this pull request, the changes of batch2 will also be merged.

**Comment by AnnaRybakovaT** [comment]
> I think you will not have to open new pull request.

Thanks! I see!

**Comment by funderburkjim** [comment]
@drdhaval2785  and @AnnaRybakovaT  
I got notified of this commit to csl-orig: https://github.com/sanskrit-lexicon/csl-orig/commit/6b45d9dadac7f7093c23a089f12ccca585ab0c3f

But when I do a git pull on csl-orig,  it does not come through .

Note the message: `This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.`

Something's not quite right.

**Comment by AnnaRybakovaT** [comment]
Dear all,
The final pull request  pe.txt  Batch 3 has done.
As well the file pe_error.txt is updated.

**Comment by gasyoun** [comment]
> The final pull request pe.txt Batch 3 has done.

Good, that the worker, slower than before, goes on.

+121 -121

**Comment by AnnaRybakovaT** [comment]
> slower than before,

That is true. My seasonal break has finished. From time to time I will try to do some work (I have the list of dictionaries), but at full speed I hope to come back in November (of cource, if everything is fine)))).

**Comment by funderburkjim** [comment]
@drdhaval2785   Are you handling the pull request, or should I?

**Comment by drdhaval2785** [comment]
All pull requests regarding PE have already been merged.

**Comment by drdhaval2785** [comment]
PUI also merged.
